extensions/amp-auto-ads/amp-auto-ads.md

```diff
@@ -77,6 +77,7 @@ should be specified on the tag.
 
 ## Supported ad networks
 - [AdSense](../../ads/google/adsense.md)
+- Denakop
```

Comment:
could you provide a link to your product page?

Reply:
Can we add the doc inside `/ads/denakop.md` ? As we don't have a `<amp-ad>` integration I'm not sure if this is the best location ?